notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1782_M2_RMP_functionality_check_small_range.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #47.        tangent_force_error = [abs(round(el, 2)) for el in tangent_force_error]
</pre><p>You can import the numpy as the np, followed by:</p><pre class="ql-syntax" spellcheck="false">tangent_force_error = np.abs(np.array(tangent_force_error)).max()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>